app/presenters/profile.rb

```diff
@@ -77,6 +77,14 @@ def mentored_tracks
         user.track_mentor
       end
 
+      def has_finished_tracks?
+        finished_tracks.any?
```

Comment:
I don't think we need this method. It's just as readable to say `finished_tracks.any?` in the view.